src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/not_found.html.twig

```diff
@@ -0,0 +1,48 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+{% extends '@PrestaShop/base.html.twig' %}
+{% block stylesheets %}
+    <link rel="stylesheet" href="{{ asset('themes/new-theme/public/theme' ~ rtl_suffix ~ '.css') }}" />
+{% endblock %}
+
+{% block body %}
+    <div class="card-block">
+        <div class="row justify-content-center">
+            <img src="{{ asset('themes/new-theme/img/empty_state/missing_combination.png') }}">
+        </div>
+      <div class="row justify-content-center combination-not-found-description">
+        <h2>{{ 'This combination doesn\'t exist anymore'|trans({}, 'Admin.Catalog.Feature') }}</h2>
+      </div>
+        <div class="row justify-content-center">
+            <button type="button" aria-label="Close" class="btn btn-primary" data-dismiss="modal">
+                <span>{{ 'Close and refresh'|trans({}, 'Admin.Catalog.Feature') }}</span>
+            </button>
+        </div>
+    </div>
+{% endblock %}
+
+{% block javascripts %}
+    <script src="{{ asset('themes/new-theme/public/combination_bulk.bundle.js') }}"></script>
+{% endblock %}
```

Comment:
With the limited content of this page I don't think you even need to include some JS here